aaf_monitor_logs/scripts/monitor_logs.py

```diff
@@ -0,0 +1,118 @@
+                                                          /task_executor/events \
+                                                          /emergency_stop_status \
+                                                          /info_terminal/active_screen \
+                                                          /info_terminal/task_outcome \
+                                                          /info_task_server/goal \
+                                                          /info_task_server/result \
+                                                          /info_task_server/cancel \
+                                                          /bellbot/goal \
+                                                          /bellbot/result \
+                                                          /bellbot/cancel \
+                                                          /walking_group_fast/goal \
+                                                          /walking_group_fast/result \
+                                                          /walking_group_fast/cancel \
+                                                          /walking_group_slow/goal \
+                                                          /walking_group_slow/result \
+                                                          /walking_group_slow/cancel \
+                                                          /store_logs/cancel \
+                                                          /store_logs/goal \
+                                                          /store_logs/result \
+                                                          /strands_webserver/display_1/page \
+                                                          /strands_webserver/display_2/page \
+                                                          /strands_webserver/display_3/page \
+                                                          /strands_webserver/display_4/page \
+                                                          /strands_webserver/display_5/page \
+                                                          /strands_webserver/display_6/page \
+                                                          /strands_webserver/display_7/page \
+                                                          /strands_webserver/display_8/page \
+                                                          /strands_webserver/display_9/page")
+        self.topic_list = self.topics_to_check.split()
+
+        self.topics_map = {}
+
+        self.pub = rospy.Publisher('monitored_logs', String, queue_size=10)
+        self.timer = rospy.Timer(rospy.Duration(30), self.timer_callback)
+
+    def timer_callback(self, time):
+        for topic in self.topic_list:
+            #print "Getting messages for " + topic
+            nbr_messages = bridge.find_nbr_entries("roslog", dc_util.topic_name_to_collection_name(topic))
```

Comment:
It should indeed be `self`. Not sure why this works but it does. I'll fix it. As far as roslog goes, the logged topic and monitored topics are all in roslog.